### ISSUE TITLE
Error when trying to start taisei: "vfs_load_packages: VFS error: No error"

### ISSUE THREAD

**Issue by HybridDog** [taisei-project/taisei#210]
I have managed to compile and install taisei on Lubuntu 18.04, but it shows an error when I try to start it and then quits when I click "Ok".
I use the latest version of taisei: current master (commit e9085a7a4d978eda1d3b19dd98f731c78aa369bd)

The error message just says that there is no error:
![2020-04-09-142231_200x100_scrot](https://user-images.githubusercontent.com/3192173/78894867-2b51e500-7a6e-11ea-8708-c0b770a6328a.png)
```
5         F: vfs_load_packages: VFS error: No error
```
The dark grey number (here 5) on the left is not always the same.

`taisei --help` works without problems.

**Comment by karmek-k** [comment]
I have the same issue on Linux Mint 19.3 Tricia x86_64. Built from source too.
```
$ meson --version
0.54.0

$ ninja --version
1.8.2

$ gcc --version
gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by Akaricchi** [comment]
Thanks for the report. I've confirmed this issue under a clean `TAISEI_STORAGE_PATH` directory. I'll fix it ASAP.

**Comment by Akaricchi** [comment]
This was a recent regression introduced in #207. Please let me know if the fix works for you.

**Comment by karmek-k** [comment]
Yeah, works now. Thanks @Akaricchi .

By the way, there were some log messages. I don't know if they are important or not. 
```
323       W: glcommon_build_shader_lang_table_fallback: Can not reliably determine all supported GLSL versions, resorting to guesswork.
811       E: parse_keyvalue_file_cb: VFS error: Node 'res/shader/global.pp' does not exist
811       E: finalize_resource: Failed to load postprocessing pipeline 'global'
```

**Comment by Akaricchi** [comment]
These are safe to ignore. Those errors are more verbose than is necessary; I'll address that in the future.

The first one complains about a missing optional OpenGL feature (available since OpenGL 4.3). Taisei will work just fine without it.

The last two are about a missing optional file that we don't ship by default; you can provide your own if you want to apply some custom post-processing effects to the image, but that's a yet undocumented feature, and not something most people would care about.
